Source/Renderer/AutomaticUniforms.js

```diff
@@ -1896,5 +1896,19 @@ var AutomaticUniforms = {
       return uniformState.ellipsoid.oneOverRadii;
     },
   }),
+
+  /**
+   * An automatic GLSL uniform that stores the camera's height above or below the ellipsoid.
+   *
+   * @alias czm_cameraHeight
+   * @glslUniform
+   */
+  czm_cameraHeight: new AutomaticUniform({
```

Comment:
There is already a czm_eyeHeight2D. Would it make sense to call this czm_eyeHeight and move it next to czm_eyeHeight2D in this file? czm_eyeHeight2D also contains height squared, not sure if that's needed for this uniform but it could be good to add for consistency.

There's also a czm_viewerPositionWC, czm_encodedCameraPositionMCHigh, and czm_encodedCameraPositionMCLow. It might be good to make the naming consistent between all of these uniforms and move them to the same area. 

Eye? Camera? Viewer?